bot/exts/fun/quack.py

```diff
@@ -0,0 +1,75 @@
+import random
+from typing import Optional
+
+import discord
+from aiohttp import client_exceptions
+from discord.ext import commands
+
+from bot.bot import Bot
+from bot.constants import Colours, NEGATIVE_REPLIES
+
+API_URL = 'https://quackstack.pythondiscord.com'
+
+
+class Quackstack(commands.Cog):
+    """Cog used for wrapping Quackstack."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @commands.command()
+    async def quack(self, ctx: commands.Context, ducktype: str = "duck", *, seed: Optional[str] = None) -> None:
+        """
+        Uses the Quackstack API to generate a random duck.
+
+        If a seed is provided, a duck is generated based on the given seed.
+        If a duck type is provided, either a duck or manduck will be generated.
```

Comment:
This makes it sound like if someone provides a duck type the command will randomly pick one of the two, not that you're supposed to send one of the two.
```suggestion
        Either "duck" or "manduck" can be provided to change the duck type generated.
```